app/assets/stylesheets/basic.scss

```diff
@@ -8,17 +8,14 @@ html {
   -webkit-font-smoothing: antialiased;
 }
 
-.background {
-  background-color: #ADD8E6;
+body {
+  height: 100%;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size, height, overflow